.buildkite/commands/run-unit-tests.sh

```diff
@@ -39,4 +42,8 @@ for file in "${results_files[@]}"; do
   fi
 done
 
+echo "--- 🧪 Copying test logs for test collector"
+mkdir buildkite-test-analytics
+cp $test_log_dir buildkite-test-analytics
```

Comment:
Just in case `$test_log_dir` ends up containing spaces or special characters (unlikely in our particular setup, but better safe than sorry)
```suggestion
cp "$test_log_dir" buildkite-test-analytics
```